assets/i18n/id.json

```diff
@@ -0,0 +1,262 @@
+    "dgc-reddit": "Komunitas Reddit", 
+    "dgc-dogeducation": "Komunitas Dogeducation di Reddit",
+    "dgc-twitter": "Twitter",
+    "dgc-getting-started": "Bagaimana cara memulai menggunakan Dogecoin", 
+    "dgc-setup-wallet": "Membuat sebuah dompet Dogecoin itu <strong>gampang banget</strong>, asli! Periksa panduan memulainya dibawah.", 
+    "dgc-edu-help": "Kalau kamu butuh bantuan, tanyakan di Reddit pada r/dogeducation! <br /> Kita adalah sekelompok orang yang akan membantu dan menjawab setiap pertanyaan yang Kamu ajukan!",
+    "dgc-footer-waiting": "<strong>Jadi tunggu apa lagi?</strong> Mulai pakai Dogecoin hari ini!", 
+    "dgc-footer-getting-started": "Memulai.", 
+    "dgc-footer-learn-how": "Pelajari bagaimana cara menggunakan Dogecoin.", 
+    "dgc-community-title": "Komunitas.", 
+    "dgc-community-participate": "Berpartisipasi dalam Komunitas.", 
+    "dgc-get-dogecoin-title": "Dapatkan Dogecoin.", 
+    "dgc-get-dogecoin-how": "Cari tau bagaimana cara mendapatkan Dogecoin.",
+    "dgc-stores-title": "Toko.", 
+    "dgc-stores-buy": "Beli dengan Dogecoin.",
+    "dgc-footer-credits": "Dibuat Oleh <a href=\"https://ummjackson.com\" target=\"_blank\">Jackson Palmer</a> & <a href=\"https://twitter.com/BillyM2k\" target=\"_blank\">Shibetoshi Nakamoto</a>. Situs Web dibuat oleh <a href=\"https://www.reddit.com/user/edswf1\" target=\"_blank\">/u/edswf1</a>. Logo dibuat oleh <a href=\"http://christinemix.com\" target=\"_blank\">Christine Ricks</a>.",
+    "dgc-footer-copyright": "© The Ðogecoin Project", 
+    "dgc-browser-modal-title": "PILIH SEBUAH DOMPET DIGITAL:", 
+    "dgc-browser-modal-online": "Dompet digital adalah cara tercepat dan termudah untuk menggunakan Dogecoin, tapi kurang aman jika disimpan pada komputer lokal mu. <br /><br />Kami merekomendasikanmu menyimpan Dogecoin dalam jumlah sedikit saja jika menggunakan dompet digital.  <br />Terdapat beberapa dompet berbasis peramban yang tersedia untuk kamu pakai:",
+    "dgc-browser-modal-dogechain": "Domet DogeChain", 
+    "dgc-browser-modal-holytransaction": "HolyTransaction", 
+    "dgc-browser-modal-blockio": "Block.io", 
+    "dgc-browser-modal-online-disclaimer": "<strong>Catatan:</strong> Dompet daring berikut tidak dimiliki dan dikelola oleh Dogecoin. Sejauh ini orang-orang di baliknya si terpercaya - tapi resiko ditanggung sendiri.", 
+    "dgc-ios-wallet-modal-important": "PENTING:", 
+    "dgc-ios-wallet-modal-view-only": "MyDoge untuk iOS hanya dapat digunakan untuk melihat isi dompet.", 
+    "dgc-ios-wallet-modal-not-send": "MyDoge <strong>tidak bisa</strong> mengirim Dogecoin. Kamu harus membuat dompet Dogecoin beneran sebelum bisa menggunakan aplikasi untuk melihat dompet Dogecoin ini.", 
+    "dgc-ios-wallet-modal-understand": "Oke Paham, Lanjut!", 
+    "dgc-ios-wallet-modal-los-gehts": "Klik tombolnya untuk melanjutkan ke halaman Appstore.", 
```

Comment:
I think we can use App Store instead of Appstore as per Wikipedia definition

https://en.wikipedia.org/wiki/App_Store_(iOS/iPadOS)